tests/jenkins/TestMessages.groovy

```diff
@@ -0,0 +1,87 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import static com.lesfurets.jenkins.unit.global.lib.LibraryConfiguration.library
+import static com.lesfurets.jenkins.unit.global.lib.ProjectSource.projectSource
+import org.junit.Before
+import org.junit.Test
+import com.lesfurets.jenkins.unit.declarative.DeclarativePipelineTest
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import com.lesfurets.jenkins.unit.MethodCall
+import static org.junit.Assert.*
+import static org.assertj.core.api.Assertions.assertThat
+import java.util.*
+
+class TestMessages extends DeclarativePipelineTest {
+
+    String sharedLibs = ''
+
+    @Override
+    @Before
+    void setUp() throws Exception {
+        scriptRoots += 'tests/jenkins/jobs'   
+
+        super.setUp()
+
+        helper.registerAllowedMethod('findFiles', [Map.class], null)
+        
+        helper.registerSharedLibrary(
+            library().name('jenkins')
+                .defaultVersion('<notNeeded>')
+                .allowOverride(true)
+                .implicit(true)
+                .targetPath('<notNeeded>')
+                .retriever(projectSource())
+                .build()
+            )
+    }
+
+    @Test
+    void testMessages() throws Exception {
+        def script = runScript("messages.groovy")
```

Comment:
This referencing another file `tests/jenkins/jobs/messages.groovy` right?  Is there a way we can make it easier to know which file is associated with this script.  I'm just think it would be difficult to traceback from a failure

Reply:
I've changed file names to match (e.g. `Messages.groovy` and `TestMessages.groovy`) and added `final String jenkinsScript = "tests/jenkins/jobs/Messages.groovy"` on top of each test class.